query_V1

```diff
@@ -58,30 +184,31 @@
 							JOIN ID_To_Artist j2 ON s.Song_ID = j2.Song_ID
 							JOIN Artist_PK ar ON ar.Artist_PK = j2.Artist_FK
 
-							WHERE 1
-							ORDER BY s.Title DESC, ar.Artist DESC");
+							WHERE s.title LIKE '%$searched%' or g.Genre LIKE '%$searched%' or ar.Artist LIKE '%$searched%' or a.Album LIKE '%$searched%' or s.Duration LIKE '%$searched%'
```

Comment:
Search is a function that has been taken from the search bar. This is used in the query as a wildcard to find where the search item is in the database.